docs-xml/manpages/vfs_virusfilter_clamav.8.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!DOCTYPE refentry PUBLIC "-//Samba-Team//DTD DocBook V4.2-Based Variant V1.0//EN" "http://www.samba.org/samba/DTD/samba-doc">
+<refentry id="vfs_virusfilter_clamav.8">
+
+<refmeta>
+	<refentrytitle>vfs_virusfilter_clamav</refentrytitle>
+	<manvolnum>8</manvolnum>
+	<refmiscinfo class="source">Samba</refmiscinfo>
+	<refmiscinfo class="manual">System Administration tools</refmiscinfo>
+	<refmiscinfo class="version">4.6</refmiscinfo>
+</refmeta>
+
+
+<refnamediv>
+	<refname>vfs_virusfilter_clamav</refname>
+	<refpurpose>On access virus scanner</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+	<cmdsynopsis>
+                <command>vfs objects = virusfilter_clamav</command>
+	</cmdsynopsis>
+</refsynopsisdiv>
+
+<refsect1>
+	<title>DESCRIPTION</title>
+
+	<para>This is a set of various Samba VFS modules to scan and filter
+        virus files on Samba file services with an anti-virus scanner.</para>
+
+	<para>This module is stackable.</para>
+
+</refsect1>
+
+<refsect1>
+	<title>OPTIONS</title>
+
+        <variablelist>
+
+                <varlistentry>
+                <term>virusfilter_clamav:socket path = PATH</term>
+                <listitem>
+                <para>Path of local socket for the virus scanner.
+                </para>
+                <para>If this option is not set, the default path /var/run/clamav/clamd.ctl.</para>
+                </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                <term>virusfilter_clamav:connect timeout = 30000</term>
+                <listitem>
+                <para>Controls how long to wait on connecting to the virus
+                scanning process before timing out. Value is in milliseconds.
+                </para>
+                <para>If this option is not set, the default is 30000.</para>
+                </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                <term>virusfilter_clamav:io timeout = 60000</term>
+                <listitem>
+                <para>Controls how long to wait on communications with the virus
+                scanning process before timing out. Value is in milliseconds.
+                </para>
+                <para>If this option is not set, the default is 60000.</para>
+                </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                <term>virusfilter_clamav:scan on open = yes</term>
+                <listitem>
+                <para>This option controls whether files are scanned on open.
+                </para>
+                <para>If this option is not set, the default is yes.</para>
+                </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                <term>virusfilter_clamav:scan on close = no</term>
+                <listitem>
+                <para>This option controls whether files are scanned on open.
```

Comment:
This was copy-pasted and must be "on close".

Reply:
I am fairly certain this was fixed in the actual commit which landed in 4.8.0 RC3 I think. Please, note all the manpages were merged into vfs_virusfilter.xml.

I am going to close this pull request.